src/app/events/page.tsx

```diff
@@ -74,7 +74,7 @@ const Events = async ({ searchParams }: { searchParams: searchPams }) => {
   return (
     <main className="pb-10 md:pl-72">
       <EventHeader />
-      <EventView events={events} params={searchParams} />
+      <EventView params={searchParams} events={events} />
```

Comment:
ok bro

Reply:
I think this just happened because I removed it and added it back lol, cause of how I was doing the composition.